packages/liblzma/meta.yaml

```diff
@@ -19,6 +19,7 @@ build:
         --disable-scripts \
         --disable-doc \
         --enable-shared=no \
+        --disable-dependency-tracking \
```

Comment:
I think it's disabled by default already? https://www.gnu.org/software/automake/manual/html_node/Dependency-Tracking.html

Reply:
I think it is not disabled by default. I can observe the log in main:

```
checking dependency style of /root/repo/emsdk/emsdk/upstream/emscripten/emcc... gcc3
```

which is changed in this PR:

```
checking dependency style of /root/repo/emsdk/emsdk/upstream/emscripten/emcc... none
```

Reply:
OK, thanks for checking